apps/condo/domains/meter/components/TabContent/Meter.tsx

```diff
@@ -0,0 +1,318 @@
+}
+
+const MetersTableContent: React.FC<MetersTableContentProps> = ({
+    filtersMeta,
+    tableColumns,
+    baseSearchQuery,
+    canManageMeters,
+    sortableProperties,
+    loading,
+}) => {
+    const intl = useIntl()
+    const CreateMeterButtonLabel = intl.formatMessage({ id: 'pages.condo.meter.index.CreateMeterButtonLabel' })
+    const SearchPlaceholder = intl.formatMessage({ id: 'filters.FullSearch' })
+    const IsActiveMessage = intl.formatMessage({ id: 'pages.condo.meter.Meter.isActive' })
+    const OutOfOrderMessage = intl.formatMessage({ id: 'pages.condo.meter.Meter.outOfOrder' })
+    const VerificationStartMessage = intl.formatMessage({ id: 'pages.condo.meter.Verification.StartDate' })
+    const VerificationEndMessage = intl.formatMessage({ id: 'pages.condo.meter.Verification.EndDate' })
+
+    const router = useRouter()
+    const { filters, offset, sorters, tab } = parseQuery(router.query)
+    const currentPageIndex = getPageIndexFromOffset(offset, DEFAULT_PAGE_SIZE)
+
+    const { filtersToWhere, sortersToSortBy } = useQueryMappers(filtersMeta, sortableProperties || SORTABLE_PROPERTIES)
+
+    const sortBy = useMemo(() => sortersToSortBy(sorters) as SortMetersBy[], [sorters, sortersToSortBy])
+
+    const [dateRange, setDateRange] = useDateRangeSearch('nextVerificationDate')
+    const [filtersAreReset, setFiltersAreReset] = useState(false)
+    const [isShowActiveMeters, setIsShowActiveMeters] = useState(true)
+    const [isShowArchivedMeters, setIsShowArchivedMeters] = useState(false)
+
+    const dateFallback = filtersAreReset ? null : DEFAULT_DATE_RANGE
+    const dateFilterValue = dateRange
+    const dateFilter = dateFilterValue ? dateFilterValue.map(el => el.toISOString()) : null
+
+    const handleDateChange = useCallback((value) => {
+        if (!value) {
+            setFiltersAreReset(true)
+            setDateRange(null)
+        } else {
+            setDateRange(value)
+        }
+    }, [setDateRange])
+
+    const handleOnClickVerificationDate = useCallback(() => {
+        if (!dateRange) setDateRange(dateFallback)
+    }, [dateFallback, dateRange, setDateRange])
+
+    const searchMetersQuery = useMemo(() => ({
+        ...filtersToWhere({
+            nextVerificationDate: dateFilter,
+            ...filters,
+        }),
+        ...(isShowActiveMeters && !isShowArchivedMeters)  ? { archiveDate: null } : {},
+        ...(isShowArchivedMeters && !isShowActiveMeters) ? { archiveDate_not: null } : {},
+        ...baseSearchQuery,
+    }), [baseSearchQuery, dateFilter, filters, filtersToWhere, isShowActiveMeters, isShowArchivedMeters])
+
+    const {
+        loading: meterReadingsLoading,
+        count: total,
+        objs: meterReadings,
+        refetch,
+    } = MeterForOrganization.useObjects({
+        sortBy,
+        where: searchMetersQuery,
+        first: DEFAULT_PAGE_SIZE,
+        skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
+    }, {
+        fetchPolicy: 'network-only',
```

Comment:
Did you check, that you're logic can work with default fetch policy? Maybe you can easily remove this line?

Reply:
you're, removed it